src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int numberRandom = random.nextInt(101);
```

Comment:
Do not mark conversation as resolved when you didn't implement required change.

![image](https://github.com/mate-academy/jv-lottery/assets/140905715/a75ce0a2-241d-440f-9828-80373f05f462)